components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/SubscriptionServices.cs

```diff
@@ -570,6 +555,23 @@ internal sealed class SubscriptionWrapper : ISubscription {
                                 results.Count(r => r != null && StatusCode.IsNotGood(r.StatusCode)),
                                 rawSubscription.DisplayName);
                         }
+                        else {
+                            _logger.Information("Now issuing ConditionRefresh for subscription " +
+                                "{subscription}", rawSubscription.DisplayName);
+                            try {
+                                rawSubscription.ConditionRefresh();
+                            }
+                            catch (ServiceResultException e) {
+                                _logger.Information("ConditionRefresh for subscription " +
+                                    "{subscription} failed with a ServiceResultException '{message}'", rawSubscription.DisplayName, e.Message);
+                            }
+                            catch (Exception e) {
+                                _logger.Information("ConditionRefresh for subscription " +
+                                    "{subscription} failed with an exception '{message}'", rawSubscription.DisplayName, e.Message);
+                            }
```

Comment:
how is the error handling done here? Just ignored?

Reply:
Changed so that I set noErrorFound = false; Seems consistent with how other errors are handled

Reply:
funny noerror logic :-)

---
In reply to: [622761857](https://github.com/Azure/Industrial-IoT/pull/1124#discussion_r622761857) [](ancestors = 622761857)

Reply:
Yes, something for another PR perhaps.